.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+## 📌 Description
+
+Please include a summary of the changes and the related issue. Provide enough context for reviewers to understand what the PR is doing.
+
+- What is the motivation for this change?
```

Comment:
can we remove this motivation part? we don't want to block people to the point where they start thinking of a story to write. 

the "what problem does it solve?" question should suffice here.

Reply:
awesome! 🍷